### PR TITLE
Workaround GitHub not supporting Flit metadata

### PR DESCRIPTION
**Pull request by sethmlarson**
Ref: https://github.com/github/feedback/discussions/6456 This method is already being implemented by Flask: https://github.com/pallets/flask/blob/main/setup.py#L3-L16

### CODE REVIEW

**Review by pquentin — APPROVED** [review]
Thanks!

**Comment by pquentin** [comment]
Can't merge this myself as I'm not a code owner for this file.

**Comment by sethmlarson** [comment]
Thanks for the review Quentin! :)

**Pull request merged**
